UnitTests/test_saveFile.py

```diff
@@ -15,17 +15,21 @@
 
 def test_SaveFile():
 
-    if (os.path.isfile("/testResults/save.rs9")):
-        os.remove("/testResults/save.rs9")
+    savePath = r"\testResults\save.rs9"
+    filepath = dirName + savePath
```

Comment:
Name of the variable should be in camelCase or snake_case. In this instance filePath.